ckeditor_uploader/views.py

```diff
@@ -110,25 +113,7 @@ def post(self, request, **kwargs):
     def _save_file(request, uploaded_file):
         filename = get_upload_filename(uploaded_file.name, request.user)
 
-        img_name, img_format = os.path.splitext(filename)
-        IMAGE_QUALITY = getattr(settings, "IMAGE_QUALITY", 60)
-
-        if (str(img_format).lower() == "png"):
```

Comment:
`os.path.splitext('mypicture.png')` would return `("mypicture", ".png")`, and not  `("mypicture", "png")` so those lines are never called. As you expect the source file and the uploaded file to have the same hash in your tests, I deleted it.

Reply:
I'll fix it

Reply:
@matthiask and @hsmett : I am currently using the upload feature and it is not working properly. 
.png files are converted to .jpg when I am reading the ckeeditorbrowse response.
I installed django-ckeditor, and the version is: 5.9.0
Please, can you guide me on this? I do need .png files because high number of our clients use transparent .png files.